example/typescript/timeline.ts

```diff
@@ -1,6 +1,7 @@
-import Mastodon from '../../src/mastodon'
+import Mastodon from 'megalodon'
+import { Status } from 'megalodon'
```

Comment:
__[TSLint]__

Multiple imports from 'megalodon' can be combined into one. (no-duplicate-imports)


<!-- Comment by [Sider](https://sider.review) -->